R/expr-expr.R

```diff
@@ -342,14 +361,16 @@ expr__gt <- function(other) {
 
 #' Check greater or equal inequality
 #'
-#' @inherit expr__add description params
+#' @inheritParams expr__eq
 #' @inherit as_polars_expr return
 #'
 #' @examples
-#' pl$lit(2) >= 2
-#' pl$lit(2) >= pl$lit(2)
-#' pl$lit(2)$gt_eq(pl$lit(2))
-expr__gt_eq <- function(other) {
+#' df <- pl$DataFrame(x = 1:3)
+#' df$with_columns( 
+#'   with_ge = pl$col("x")$ge(pl$lit(2)),
+#'   with_symbol = pl$col("x") >= pl$lit(2)
+#' )
+expr__ge <- function(other) {
```

Comment:
`gt_eq()` exists in rust-polars but not in the py-polars API (also more `ne`, `ne_missing`, `lt_eq`). Same thing as for the div operations, I think we should follow the py-polars API.